rpcs3/rpcs3.appdata.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2018 RPCS3-->
+​<component type="desktop-application">
+    ​<id>RPCS3</id>
+    ​<metadata_license>FSFAP</metadata_license>
+    ​<project_license>GPL-2.0</project_license>
+    ​<name>Power Statistics</name>
```

Comment:
Copypasta from the freedesktop.org example? The name and summary tags are incorrect I believe

Reply:
Oops, lol, thanks for pointing that out.